src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: rectangle, area: " + (firstSide * secondSide)
+                + " sq.units, firstSide: " + firstSide + " units, secondSide: "
+                + secondSide + " units, color: " + new ColorSupplier().getRandomColor();
```

Comment:
Think about which variables should be local in the method and which should be class-level